src/app/admin/register-admin/register-admin.component.ts

```diff
@@ -113,7 +113,7 @@ export class RegisterAdminComponent implements OnInit {
             }
             if (xhr.status === 200) {
               debugger;
-              if (!isNullOrUndefined(xhr.response.user)) {
+              if (!isNullOrUndefined(xhr.response.success)) {
                 this.toastr.success('User Created !', '', { positionClass: 'toast-top-right', timeOut: 5000 });
                 this.resetForm(form);
```

Comment:
Why this message is hard coded and not coming from some resource file or translation file?